spec/stepmod/converters/strong_spec.rb

```diff
@@ -69,7 +69,7 @@
 
           [stem]
           ++++
-          χ bb(My node)_{ms} = bb(V - E + 2F - L)_{l}bb(- 2(S - G)^{s}) = 0
+          χ My node_{ms} = V - E + 2F - L_{l} - 2(S - G^{s}) = 0
```

Comment:
Here the `bb(My node)_{ms}` should not just be `My node_{ms}` because the "My" will no longer be combined with "My node"?

Reply:
@ronaldtse This was an existing issue before this change. This PR is for only reverting the bold and italics change. 

I've opened a [new issue](https://github.com/metanorma/stepmod-utils/issues/193) for that and will fix this in a separate PR.